diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index cb7b82021653212a0c4f36a77f7a131630f6fe27..0000000000000000000000000000000000000000
diff --git a/esm/index.js b/esm/index.js
index 7e6b46912faec6e577311c626683c97211ed34fb..d148f0cf248a46e775304c2c33d906e8706ad5d1 100644
--- a/esm/index.js
+++ b/esm/index.js
@@ -6,6 +6,7 @@ export { ono };
 export default ono;
 // CommonJS default export hack
 if (typeof module === "object" && typeof module.exports === "object") {
-    module.exports = Object.assign(module.exports.default, module.exports);
+  module.exports.default = module.exports.default || {}
+  module.exports = Object.assign(module.exports.default, module.exports); 
 }
 //# sourceMappingURL=index.js.map